executor/aggfuncs/func_count.go

```diff
@@ -0,0 +1,380 @@
+package aggfuncs
+
+import (
+	"encoding/binary"
+	"github.com/juju/errors"
+	"unsafe"
+
+	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/types/json"
+	"github.com/pingcap/tidb/util/chunk"
+)
```

Comment:
actually, this is what I'm trying to explain:
```go
import (
	"encoding/binary"
	"unsafe"
 
	"github.com/juju/errors"
	"github.com/pingcap/tidb/expression"
	"github.com/pingcap/tidb/sessionctx"
	"github.com/pingcap/tidb/types"
	"github.com/pingcap/tidb/types/json"
	"github.com/pingcap/tidb/util/chunk"
)
```

Reply:
Yes, I got the idea just now.